Tests/Editor/BuildPlayerTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.IO;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Tests
+{
+	[InitializeOnLoad]
+	public class BuildPlayerTest
+	{
+		const string kTestPath = "_Test";
+		const string kVerifySavingAssets = "VerifySavingAssets";
+
+		bool? m_VerifySavingAssets;
+
+		[SetUp]
+		public void SetUp()
+		{
+			if (EditorPrefs.HasKey(kVerifySavingAssets))
```

Comment:
I tried this again and did not get prompted. I tried with the setting enabled before running tests, and disabled beforehand. I tried the tests individually as well as in a batch.